packages/blockchain-wallet-v4-frontend/src/modals/BuySell/PaymentMethods/Methods/BankWire/index.tsx

```diff
@@ -36,26 +38,22 @@ const BankWire: React.FC<Props> = ({ icon, onClick, text, value }) => (
     <DisplayIconBank>{icon}</DisplayIconBank>
     <Content>
       <DisplayTitleBank>{text}</DisplayTitleBank>
-      <DisplaySubTitle>
-        {value.currency === 'ARS' ? (
+      <DisplaySubTitle>{getSubTitleByCurrency(value.currency)}</DisplaySubTitle>
+      <Description>
+        {value.currency === 'EUR' ? (
```

Comment:
Here it is the same thing as with the other messages. There are 4 nested ternaries and it is hard to track and not very clear